hack/make-rules/test-cmd.sh

```diff
@@ -34,7 +34,7 @@ function run_kube_apiserver() {
   kube::log::status "Starting kube-apiserver"
 
   # Admission Controllers to invoke prior to persisting objects in cluster
-  ADMISSION_CONTROL="Initializers,NamespaceLifecycle,LimitRanger,ResourceQuota"
+  ENABLE_ADMISSION_PLUGINS="Initializers,NamespaceLifecycle,LimitRanger,ResourceQuota"
```

Comment:
I am still confused: we do not use --admission-control anymore, i.e. the default plugins are enabled. Why do we need these in addition? Are they disabled by default? If yes, maybe we should remove them from  `ENABLE_ADMISSION_PLUGINS` as well. This looks more like a drift from the default in the Go code.

Reply:
Except `NamespaceLifecycle` all others are disabled default.

Reply:
Are these the plugins for integration tests? Didn't notice that we left the local cluster file here.

Reply:
I think so.

Reply:
Removed `NamespaceLifecycle` here.